src/bundle/Resources/views/content/widgets/content_edit.html.twig

```diff
@@ -1,6 +1,6 @@
 {% trans_default_domain 'locationview' %}
 
-<div class="ez-extra-actions ez-extra-actions--edit ez-extra-actions--hidden bg-secondary" data-actions="edit">
+<div class="ez-extra-actions ez-extra-actions--edit ez-extra-actions--hidden bg-secondary {% if form.language.vars.choices|length == 1 %} ez-extra-actions--prevent-show {% endif %}" data-actions="edit">
```

Comment:
I'm wondering, when can be a situation that a `div` has a class of `ez-extra-actions--edit` but it's action is not to edit? Maybe you could drop `data-actions="edit"` attribute?

Reply:
I'm also wondering, if you really need this class `ez-extra-actions--prevent-show`?

Reply:
Don't know, but identical thing is in create button so prolly, there are some js that depends on it. Even in extra.action.js there is selector with that.